Cargo.toml

```diff
@@ -37,6 +37,7 @@ serde_json = "1.0.109"
 once_cell = "1.8.0"
 actix-multipart = "0.6.1"
 parking_lot = "0.12.3"
+percent-encoding = "2.3"
```

Comment:
Hey @tqq1994516 👋 

Apologies for this very very late revert. I am starting the review now.


I don't see this crate being used anywhere. Do we need it?

Reply:
Okay, nvm